src/System.Text.Formatting.Globalization/System.Text.Formatting.Globalization.csproj

```diff
@@ -17,6 +17,7 @@
     <PackageProjectUrl>https://github.com/dotnet/corefxlab</PackageProjectUrl>
     <PackageLicenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</PackageLicenseUrl>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <RootNamespace /> 
```

Comment:
The tests fail if I remove this. I grabbed this change from here: PR #1171 

The same applies in other projects that use resx.

Failed   System.Text.Formatting.Globalization.Tests.NonInvariantCultureTests.CustomCulture
Error Message:
 System.Exception : resource missing
Stack Trace:
   at System.Text.Formatting.EncodingProvider.CreateEncoding(String localeId) in C:\Users\ahkha\Documents\GitHub\Fork\corefxlab\src\System.Text.Formatting.Globalization\System\Text\Formatting\FormattingDataProvider.cs:line 27
   at System.Text.Formatting.Globalization.Tests.NonInvariantCultureTests.CustomCulture() in C:\Users\ahkha\Documents\GitHub\Fork\corefxlab\tests\System.Text.Formatting.Globalization.Tests\BasicTests.cs:line 16